src/ucp/api/ucp_def.h

```diff
@@ -771,5 +771,45 @@ typedef struct ucp_ep_params {
  */
 #define UCP_ENTITY_NAME_MAX 32
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief A transport name and device name pair used by this endpoint.
+ */
+typedef struct ucp_transport_entry_t {
+    /**
+     * The name of a transport layer used by this endpoint.
+     */
+    const char *transport_name;
```

Comment:
what is the lifetime of the name string returned here?

Reply:
From your previous comments I understand the lifetime of the transport_name and device_name strings to be as long as the endpoint is active/valid. Once the endpoint is shut down, there is no expectation for these strings to be valid. A caller that wants to keep the data after the endpoint is shutdown has to make a copy.

Reply:
i think would be good to write this in the doc